src/routes/templates/templates.json

```
@@ -766,5 +766,20 @@
 		"category": "SvelteKit",
 		"tags": ["blog", "markdown", "templates"],
 		"stars": 4
+	},
+	{
+		"title": "Urara",
+		"url": "https://github.com/importantimport/urara",
+		"description": "Sweet & Powerful SvelteKit Blog Template.",
+		"npm": "",
+		"addedOn": "2022-03-09",
+		"category": "SvelteKit",
+		"tags": [
+			"blog",
+			"mdsvex",
+			"typescript",
+			"templates"
+		],
```

The format wanted by Prettier is the following:

```suggestion
		"tags": ["blog", "mdsvex", "typescript", "templates"],
```

----

You can run `npm run format` to let Prettier fix it for you